stories/notification/notification.stories.js

```diff
@@ -5,512 +5,295 @@ import '../../dist/notification.css';
-        <div class="fd-notification__title">This is the notifications title</div>
-        <button aria-label="decline-button" role="button" class="fd-button fd-button--transparent fd-notification__close">
-            <i class="sap-icon--decline"></i>
-        </button>
+export const warning = () => `
+<div class="fd-notification">
+    <div class="fd-message-strip fd-message-strip--warning fd-message-strip--dismissible" role="alert">
+      <p class="fd-message-strip__text">
+        Vivamus sagittis diam in vehicula lobortis sapien arcu mattis erat vel aliquet.
+      </p>
+      <button class="fd-button fd-button--transparent fd-button--compact fd-message-strip__close" aria-label="Close">
+      </button>
     </div>
-    <div class="fd-notification__body">
+    <div class="fd-notification__body fd-notification__body--message">
+       <span class="fd-avatar fd-avatar--xs fd-avatar--circle fd-avatar--thumbnail" style="background-image: url('http://lorempixel.com/400/400/nature/4/')" role="presentation" aria-label="John Doe"></span>
         <div class="fd-notification__content">
-            <div class="fd-notification__avatar">
-                <span class="fd-avatar fd-avatar--xs fd-has-background-color-accent-9" aria-label="Wendy Wallace">WW</span>
-            </div>
-            <div class="fd-notification__text">
-                <div class="fd-notification__description">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-                </div>
-                <div class="fd-notification__metadata">
-                    Excepteur sint occaecat cupidatat non proident.
-                </div>
+            <div class="fd-notification__header">
+              <h2>You have new items</h2>
             </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+            <p>Tom &middot; 7 minutes ago</p>
         </div>
-        <div class="fd-notification__footer">
-            <div class="fd-notification__actions">
-                <button role="button" class="fd-button fd-button--positive">Approve</button>
-                <button role="button" class="fd-button fd-button--negative">Reject</button>
-            </div>
+        <div class="fd-notification__actions">
+            <button class="fd-button fd-button--compact">Open</button>
+            <button class="fd-button fd-button--transparent fd-button--compact" aria-label="Close">
+                <i class="sap-icon--decline"></i>
+            </button>
         </div>
     </div>
 </div>
 `;
-error.parameters = {
+warning.parameters = {
```

Comment:
Notifications can display warning alerts by adding the `fd-message strip fd-message-strip--warning fd-message-strip--dismissible` class together with `role="alert"`.

Reply:
The description for **Information** is duplicated here, make sure to replace with the one I've written for **Warning**.